src/doxygen.cpp

```diff
@@ -11396,6 +11397,47 @@ void adjustConfiguration()
       }
     }
   }
+  // create input file exncodings
+
+  // check INPUT_ENCODING
+  void *cd = portable_iconv_open("UTF-8",Config_getString(INPUT_ENCODING).data());
+  if (cd==reinterpret_cast<void *>(-1))
+  {
+    term("unsupported character conversion: '%s'->'%s': %s\n"
+        "Check the 'INPUT_ENCODING' setting in the config file!\n",
+        qPrint(Config_getString(INPUT_ENCODING)),qPrint("UTF-8"),strerror(errno));
+  }
+
+  // check and split INPUT_FILE_ENCODING
+  const StringVector &fileEncod = Config_getList(INPUT_FILE_ENCODING);
+  bool encodingOK = true;
+  for (const auto &mapping : fileEncod)
+  {
+    QCString mapStr = mapping.c_str();
+    int i=mapStr.find('=');
+    if (i==-1)
+    {
+      continue;
+    }
+    else
+    {
+      QCString pattern = mapStr.left(i).stripWhiteSpace().lower();
+      QCString encoding = mapStr.mid(i+1).stripWhiteSpace().lower();
+      if (pattern.isEmpty() || encoding.isEmpty())
+      {
+        continue;
+      }
+      void *cd = portable_iconv_open("UTF-8",encoding.data());
```

Comment:
I'm missing the corresponding portable_iconv_close() or was there a reason to omit it?

Reply:
It was just forgotten, I've corrected it.